service/history/historyEngine3_eventsv2_test.go

```diff
@@ -252,6 +256,7 @@ func (s *engine3Suite) TestStartWorkflowExecution_BrandNew() {
 			Identity:                            identity,
 			RequestID:                           requestID,
 		},
+		PartitionConfig: partitionConfig,
```

Comment:
I would not fill this in immediately unless we're sure it's going to be present in all cases.

Reply:
the change is backward compatible, so previous tests covered the case when it's nil